src/Atoms/Leaderboard/SummaryLine.js

```diff
@@ -0,0 +1,37 @@
+import SummaryLineStyled from './SummaryLineStyled';
+import { penaltyScore } from './penaltyScoreCalc';
+import propTypes from 'prop-types';
+
+const SummaryLine = ({ result }) => {
+  const scoresArr = result.map((score) => {
+    return score.score;
+  });
+  const score = scoresArr.reduce((accu, val) => {
+    return accu + val;
+  });
+
+  const isDisqualified = result.some((element) => element.disqualified);
+  // points aquired by a dog in all excersizes/tests at one selected class
+  function pointsAquired() {
+    if (score < 5 && score > 1) {
+      return 'pointsAquired.';
```

Comment:
![image](https://user-images.githubusercontent.com/92270179/150881171-e2e2ab87-5427-479e-954a-3015d03be00f.png)
should be "punkty"?

Reply:
Nice - you tested different values! Its a result of "change all occurences" in use - the function was named "punkty" before :)